pkg/crc/cluster/cluster.go

```diff
@@ -126,7 +126,7 @@ func StopAndRemovePodsInVM(sshRunner *ssh.SSHRunner) error {
 	// the pods are not completely stopped and when remove happens it will throw
 	// an error like below.
 	// remove /var/run/containers/storage/overlay-containers/97e5858e610afc9f71d145b1a7bd5ad930e537ccae79969ae256636f7fb7e77c/userdata/shm: device or resource busy
-	stopAndRemovePodsCmd := `bash -c 'sudo crictl stopp $(sudo crictl pods -q) && sudo crictl rmp $(sudo crictl pods -q)'`
+	stopAndRemovePodsCmd := `bash -c 'export pods=$(sudo crictl pods -q) && sudo --preserve-env=pods crictl stopp ${pods} && sudo --preserve-env=pods crictl rmp ${pods}'`
```

Comment:
now that I see this again, I wonder if `sudo bash -c `export pods=$(crictl pods -q) && crictl stopp ${pods} && crictl rmp ${pods}'` would work.

Reply:
@cfergeau if you tested this then I am OK with this change but since now I have remove all other commits which use to test it on the CI so I would go with this only.

Reply:
Nope I did not test it, which is why it's merely a random suggestion. I can run a few tests though while I'm doing reviews.